open_dread_rando/missile_color_patcher.py

```diff
@@ -0,0 +1,43 @@
+from copy import deepcopy
+from enum import Enum
+
+from open_dread_rando.patcher_editor import PatcherEditor
+
+from mercury_engine_data_structures.formats.bsmat import Bsmat
+
+# model names for pride missiles
+class MissileVariant(Enum):
```

Comment:
I'm not sure enums are the best case for this. Could have a dataclass with the mat_name/shader_path/rgba fields, and a global dict of all instances